patches/third_party-blink-renderer-core-frame-reporting_observer.idl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/core/frame/reporting_observer.idl b/third_party/blink/renderer/core/frame/reporting_observer.idl
+index a248a08fcd1e3566c838f26393fd64c877bab11b..561143b7c22dcfa00f734d0146036ce37013ab44 100644
+--- a/third_party/blink/renderer/core/frame/reporting_observer.idl
++++ b/third_party/blink/renderer/core/frame/reporting_observer.idl
+@@ -7,7 +7,7 @@
+ callback ReportingObserverCallback = void (sequence<Report> reports, ReportingObserver observer);
+ 
+ [
+-    Constructor(ReportingObserverCallback callback, optional ReportingObserverOptions options),
```

Comment:
I'm not sure if this needs to happen here, can't we block this in the network stack?

Reply:
does this actually make a callback directly in the renderer without going back to the browser process? I think there was browser process code for this?

Reply:
@bridiver - The network calls are blocked by disabling `background sync`. The callback exposes the `ViolationReports` to the javascript. Callbacks are handled in the renderer here: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/frame/reporting_observer.cc?l=38